app/controllers/groups_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
         flash[:notice] = "Group created successfully!"
         format.html { redirect_to groups_path }
       else
+        flash[:error] = "#{ @group.errors.messages.first[0] } #{ @group.errors.messages.first[1][0] }"
```

Comment:
why need this ? "#{ @group.errors.messages.first[0] } #{ @group.errors.messages.first[1][0] }"
can we use @group.errors.full_messages ?